CHANGES.rst

```diff
@@ -4,3 +4,8 @@ CHANGE LOG
 0.0.1
 -----
 - initial release
+
+1.0.0
```

Comment:
@Mitso @cblignaut we usually add the change logs in the release process, it's ok now but in feature lets update the change log in the release branch